### DISCUSSION TITLE
When is the next release?

### DISCUSSION THREAD

**Discussion by Dagur** [jesseduffield/lazygit#3191] [General]
I hope you don't mind me asking. I've noticed that a lot of work has been done since 0.40.2 so I'm excited to see some of the changes :)

**Comment by mark2185** [comment]
Soon-ish™ :)

The plan was after the holidays to minimize the risk of a faulty release during "downtime", so probably in the next couple of weeks.

**Comment by jlanzarotta** [comment]
Any news on a soon-ish release ;)

**Comment by hasecilu** [comment]
https://github.com/jesseduffield/lazygit/releases/tag/v0.41.0

**Comment by edhgoose** [comment]
I hope you don't mind me resurrecting an old thread, but I wondered if there might be another release soon? Thank you for your work!

**Reply by skoch13** [reply]
I assume that once https://github.com/jesseduffield/lazygit/pull/4146 is settled, releases will be more frequent.

**Reply by jesseduffield** [reply]
@edhgoose we've cut a new release just now (https://github.com/jesseduffield/lazygit/releases/tag/v0.45.0). And https://github.com/jesseduffield/lazygit/pull/4146 is now merged so we can expect monthly releases from now on

**Reply by edhgoose** [reply]
Awesome, thank you so much! Can't wait to try it out. Looking forward to the monthly releases too.
